Сетка страниц и крупных блоков на флексах

nikolai-skor opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/35/user/2178613/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/2178613-sedona-35/4/

nikolai-skor committed:
Добавляет липкий "футер" и сетку крупных блоков на флексах.

Dima-Dim commented on catalog.html:
```diff
@@ -44,13 +44,15 @@
     <!-- main -->
-    <main class="main-catalog">
+    <main class="main-catalog main-container">
+      <div class="page-container">
```
Что делает это `div` чего не может сделать `main`?

Dima-Dim commented on catalog.html:
```diff
@@ -44,13 +44,15 @@
-    <main class="main-catalog">
+    <main class="main-catalog main-container">
+      <div class="page-container">
+      <div class="catalog-filter-wrapper">
```
Тебе самому понятно что во что вложено с такими отступами?

Dima-Dim reviewed (changes requested):
Сейчас крупные блоки уже должны быть на своих, но блоки накладываются друг на друга. Это говорит о том, что сетка не выстроена.

nikolai-skor committed:
Форматирует код.

Форматирует HTML код (в том числе отступы <div>).  По замечанию " <div class="page-container">"-на лайве объяснили следующим образом: "При добавлении новых элементов в будущем наша верстка не сломается." Иду  последовательно по программе, лишнее устраняю/редактирую по ходу продвижения на обучающем курсе.

nikolai-skor committed:
Исправляет визуальную накладку блоков.

nikolai-skor committed:
Форматирует html без закрытия одиночных тегов

Dima-Dim commented on index.html:
```diff
@@ -50,101 +48,113 @@
+              <h3 class="advantages-title">Мало туристов</h3>
+              <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+            </li>
+            <img class="advantages-image" src="images/pictures/advantages-bridge.jpg" width="800" height="384"
```
validator.w3.org/nu

Dima-Dim commented on index.html:
```diff
@@ -50,101 +48,113 @@
+          <ul class="social-list">
+            <li class="social-item">
+              <h3 class="visually-hidden">Вконтакте</h3>
+              <a href="https://vk.com/htmlacademy">
```
https://up.htmlacademy.ru/htmlcss/35/criteries#ALLY-02

Dima-Dim replied:
А тут что?

Dima-Dim commented on styles/styles.css:
```diff
@@ -307,6 +512,12 @@ list-style-type: none;
   text-transform: uppercase;
   text-decoration: none;
   color: #FFFFFF;
+  background-color: #7DB54F;;
```
Одна `;` лишняя.

Dima-Dim reviewed (changes requested):
Проверить валидатором validator.w3.org/nu и внести правки.

nikolai-skor committed:
Исправляет код в соответствии с критериями, устраняет мелкие недочеты.

Dima-Dim commented on index.html:
```diff
@@ -50,101 +48,115 @@
+          <h2 class="visually-hidden">Социальные сети</h2>
+          <ul class="social-list">
+            <li class="social-item">
+              <h3 class="visually-hidden">Вконтакте</h3>
```
Чем это должно помочь? И почему заголовок? Можно же посмотреть как это сделано в демках?

Dima-Dim commented on index.html:
```diff
@@ -50,101 +48,115 @@
+            </li>
+          </ul>
+        </section>
+        <h3 class="visually-hidden">Позвонить.</h3>
```
Тоже непонятное появление?

Dima-Dim reviewed (commented):
Поправь оставшиеся замечания в следующих заданиях.
